datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaTransportProvider.java

```diff
@@ -119,7 +119,7 @@ public KafkaTransportProvider(DatastreamTask datastreamTask, List<KafkaProducerW
       // If the partition is not specified. We use the partitionKey as the key. Kafka will use the hash of that
       // to determine the partition. If partitionKey does not exist, use the key value.
       keyValue = record.getPartitionKey().isPresent()
-          ? record.getPartitionKey().get().getBytes(StandardCharsets.UTF_8) : null;
+              ? record.getPartitionKey().get().getBytes(StandardCharsets.UTF_8) : keyValue;
```

Comment:
I understand you're trying to minimize changes but I think the code would be slightly clearer/better if we avoid the redundant assignment, i.e.

```java
if (record.getPartitionKey().isPresent()) {
    keyValue = record.getPartitionKey().get().getBytes(StandardCharsets.UTF_8);
}
```

Reply:
Thanks for the review @ahmedahamid. I agree with your feedback. 
I was trying to make the code look as it was prior to this [commit](https://github.com/linkedin/brooklin/commit/0230f364732d1cafc779c420108067e359fd88f7).

Let me know if you feel strongly about the code style. I can make that change and run a quick round of tests.

Reply:
Thanks for your contribution, @sdomalap